synapse/event_auth.py

```diff
@@ -637,6 +671,66 @@ def get_user_power_level(user_id: str, auth_events: StateMap[EventBase]) -> int:
             return 0
 
 
+def get_users_which_can_issue_invite(auth_events: StateMap[EventBase]) -> List[str]:
+    """
+    Return the list of users which can issue invites.
+
+    This is done by exploring the joined users and comparing their power levels
+    to the necessyar power level to issue an invite.
+
+    Args:
+        auth_events: state in force at this point in the room
+
+    Returns:
+        The users which can issue invites.
+    """
+    invite_level = _get_named_level(auth_events, "invite", 50)
```

Comment:
Other places seem to default this to `0` if it doesn't exist, but the spec says `50`.

Reply:
I think that's a spec bug.

Reply:
And I don't think we should be using different defaults to other places in this file, as that feels like its going to be *very* confusing

Reply:
I think we already might use different defaults in different places? I'll file a spec bug I suppose. 😄 

(This is partially why I wanted to abstract the power-level stuff out so we only specify the default once...)

Reply:
I don't think we do use different defaults in the auth rules check? We may as where (let's not confuse the default power levels sent when creating the room).

Reply:
Oh, we don't seem to...except here (which is the one I added).

Reply:
I updated this and filed matrix-org/matrix-doc#3281.